EvolutionModel/Aflegstapel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Evolution.Model {
+  public class Aflegstapel {
+    private Stack<Card> cards;
+
+    public Aflegstapel() {
+      cards = new Stack<Card>(129);
+    }
+
+    public int Count => cards.Count;
+
+    public void Add(Card kaart) {
+      cards.Push(kaart);
+    }
+    
+/*
+ * Er zijn twee mogelijkheden, of je verplaatst alle kaarten naar het deck, of je refresht het deck en leegt de aflegstapel. 
```

Comment:
We willen de kaarten verplaatsen. Het deck moet namelijk diezelfde kaarten krijgen die in de aflegstapel liggen. Niet meer, niet minder. De kaarten die in het spel zijn blijven daar en moeten niet gedupliceerd worden.

Het deck weet niet welke kaarten er in de aflegstapel zitten.